### ISSUE TITLE
Which dataset do the 3551 to 3555 images come from?

### ISSUE THREAD

**Issue by zpc-666** [MingSun-Tse/ASSL#7]
Hi, thanks for your excellent work. Which dataset do the 3551 to 3555 images used for validation during training come from? DIV2K 900 + Flickr2K 2650 = DF2K 3550，so do you use Set5 for validation during training ?

**Comment by MingSun-Tse** [comment]
Hi @zpc-666 , thanks for your interest in our work! Yes, 3551 - 3555 means the 5 images from Set5, which is used to tell us if the loss normally drops during training. Note, it looks like a validation set but we do not use it to select the final model.

**Comment by zpc-666** [comment]
Thanks for your reply!

> Hi @zpc-666 , thanks for your interest in our work! Yes, 3551 - 3555 means the 5 images from Set5, which is used to tell us if the loss normally drops during training. Note, it looks like a validation set but we do not use it to select the final model.
